### PR TITLE
Portfolio: desktop version #1

### PR DESCRIPTION
**Pull request by obibaadoma**
# Project Styling Requirements

## CSS

- Create a dedicated CSS file to manage your project's styles.

- Implement a mobile-first approach by setting up basic styles for elements to ensure a responsive design that starts from mobile devices.

- Utilize CSS Grid to create a structured grid-based layout for organizing your web elements effectively.

- Apply Flexbox techniques to handle the alignment and positioning of elements within their respective grid areas, enhancing the layout's flexibility.

- Utilize flex properties such as `justify-content` and `align-items` to precisely control horizontal and vertical alignment, respectively, ensuring visual consistency.

- Fine-tune your styles as necessary, making adjustments to spacing, typography, colors, and other visual aspects to harmonize with the selected design template.

By adhering to these styling requirements, you will create a visually appealing and responsive web interface while maintaining code organization in a dedicated CSS file.

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi @obibaadoma , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a new PR for re-review. Kindly submit the old PR with the new changes so that reviews can reference earlier comments or You can include a link to the old PR in the current PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html
Comment (Ofarouq310) [inline index.html:1]: - [ ] Kindly make sure for every `img` tag you have included, you should add an `alt` attribute for them, that is to reach the best semantic structure possible.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,2456 @@
...
+  display: flex;
+  flex-shrink: 0;
+  font-family: "Poppins", sans-serif;
+  font-size: 2rem;
+  font-weight: 500;
+  line-height: 1.2;
+  margin-right: 48.1rem;
+  white-space: nowrap;
+} */
+
```
Comment (Ofarouq310) [inline style.css:540]: - [ ] Kindly remove this commented piece of code, as it hurts the overall organization and professionalism of your code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+          name="comment" 
+          id="comment" 
+          required
+        ></textarea>
+        <label class="labels" for="comment">Comment</label>
+        <button type="submit" class="button-form">Get in touch</button>
+      </form>
+    </div>
+  </div>
+</section>
```
Comment (Ofarouq310) [inline index.html:256]: - [ ] As per requirements to stick to the `Figma` design as much as possible, kindly refer back to the template to use the same fonts as per the specs.

 | Your design | Figma Design |
| ------------- | ------------- |
| Screenshot 2023-06-10 214723 | image |

File: index.html [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+    <li class="dropdowns">
+      <a>Frameworks</a>
+      <button type="submit" ><img src="images\Disabled.png" alt="Dropdown button"></button>
+    </li>
+    <li class="dropdowns">
+      <a>Skills</a>
+      <button type="submit"><img src="images\Disabled.png" alt="Dropdown button"></button>
+    </li>
+  </ul>
+</section>
```
Comment (Ofarouq310) [inline index.html:212]: - [ ] As per requirements to stick to the `Figma` design as much as possible, kindly refer back to the template to use the same fonts as per the specs.

 | Your design | Figma Design |
| ------------- | ------------- |
| Screenshot 2023-06-10 214714 | Screenshot 2023-06-10 224420 |

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @obibaadoma, ✋  

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥈 

giphy

## Highlights
✅ Followed GitHub flow.
✅ Descriptive PR title & summary.
✅ No linters errors.


## Required Changes ♻️

- [ ] Everything looks good, but I believe you have oversized some of your elements, take a look at the pictures, everything looks bigger than it should, so refer back to the Figma template to see how everything looks like, kindly address this issue to make your website looks more normal.
Screenshot 2023-06-10 214647
Screenshot 2023-06-10 214656
Screenshot 2023-06-10 214723


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (obibaadoma) [reply style.css:540]: fixed

File: index.html (hunk 3 above)
Comment (obibaadoma) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (obibaadoma) [reply index.html]: done

File: index.html
Comment (obibaadoma) [reply index.html:1]: well noted and fixed

**Review by Ofarouq310 — APPROVED** [review]
Hi @obibaadoma, ✋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥇 

giphy

## Highlights
✅ Followed GitHub Workflow.
✅ Descriptive PR title & summary.
✅ No linters errors.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**